imports.go.tmpl

```diff
@@ -106,6 +108,16 @@ import (
 {{- if eq $opts.json "jsoniter" }}
 
 var json = jsoniter.ConfigCompatibleWithStandardLibrary
+
+{{- else if eq $opts.json "sonic" }}
+
+// Opinionated config for -json=sonic, see https://github.com/bytedance/sonic/blob/main/api.go.
+var json = sonic.Config{
+	NoNullSliceOrMap: true, // Encode empty Array or Object as '[]' or '{}' instead of 'null'.
+	CompactMarshaler: true,
+	CopyString : true,
+	ValidateString : true,
+}.Froze()
```

Comment:
Please see all Config options at
https://github.com/bytedance/sonic/blob/main/api.go#L37-L97

Reply:
I didn't enable these two options:
- `EscapeHTML: false` - don't escape HTML, which is only needed to safely embed JSON inside HTML `<script>` tags
- `SortMapKeys: false` - don't sort map keys, as that speeds up performance significantly -- up for discussion